sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_constants.py

```diff
@@ -7,13 +7,13 @@
 class EvaluationMetrics:
     """Metrics for model evaluation."""
 
-    GPT_GROUNDEDNESS = "gpt_groundedness"
-    GPT_RELEVANCE = "gpt_relevance"
-    GPT_COHERENCE = "gpt_coherence"
-    GPT_FLUENCY = "gpt_fluency"
-    GPT_SIMILARITY = "gpt_similarity"
+    GROUNDEDNESS = "groundedness"
+    RELEVANCE = "relevance"
+    COHERENCE = "coherence"
+    FLUENCY = "fluency"
+    SIMILARITY = "similarity"
     F1_SCORE = "f1_score"
-    GPT_RETRIEVAL_SCORE = "gpt_retrieval_score"
+    RETRIEVAL_SCORE = "retrieval_score"
```

Comment:
Retrieval should be `retrieval` instead of `retrieval_score`. However, I don't think these constants are being used by anywhere. Maybe we should consider just removing them to keep the code clean.

Reply:
I'll remove them.